examples/app_commands/info.py

```diff
@@ -1,45 +1,40 @@
+# Imports
```

Comment:
```suggestion
```

The fact that this is an import is pretty obvious and the comment isn't necessary.